### ISSUE TITLE
Picture-entity card should not raise any errors when the entity does not support turn_on/turn_off

### ISSUE THREAD

**Issue by arsaboo** [home-assistant/ui-schema#68]
I created a horizontal stack of picture-entity cards to indicate presence using the following config (I am 0.73.0.b2):
```yaml
      - type: horizontal-stack
        cards:
          - type: picture-entity
            entity: device_tracker.meta_alok
            state_image:
              "home": /local/icons/alok_home.png
              "not_home": /local/icons/alok_not_home.png
              "buckhead": /local/icons/alok_not_home.png
              "Buckhead": /local/icons/alok_not_home.png
            title: " "
          - type: picture-entity
            entity: device_tracker.meta_rashmi
            state_image:
              "home": /local/icons/rashmi_home.png
              "not_home": /local/icons/rashmi_not_home.png
            title: " "
```
I get the desired output. However, a small issue is that when I click the cards, HA tries to toggle the entity. However, given that `device_tracker` do not support turn_on/turn_off services,  I get a warning:
```
Log Details (WARNING)
Mon Jul 02 2018 15:49:07 GMT-0400 (Eastern Daylight Time)

Unable to find service device_tracker/turn_off
```

Ideally, we should only toggle supported entities.
